test_nb.ipynb

```diff
@@ -2,12 +2,13 @@
  "cells": [
```

Comment:
<p>Would be informative to add number of annotators per item.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/justinek/review_notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>